### PR TITLE
[INTEL MKL] Fix for build error.

### PR DESCRIPTION
**Pull request by agramesh1**
Fix for build errors caused by the refactoring commit https://github.com/tensorflow/tensorflow/commit/8ba31084528f63e623fb5f58d9ee09b148088c6d

### CODE REVIEW

**Review by alextp — DISMISSED** [review]
Ouch

**Comment by agramesh1** [comment]
@gbaned  The import/copybara test is failing, can you provide more on the failure?

**Comment by agramesh1** [comment]
@gbaned  @alextp  the previous commit was causing the Windows build test  to fail due to circular dependencies.  I updated PR with a new commit that will resolve the issue.  Please review.

**Review by alextp — APPROVED** [review]

**Comment by agramesh1** [comment]
Hi @gbaned . It looks like the Windows Bazel GPU failure is not related to the changes made in the PR.   Can you restart the tests to see if it fails again?

**Comment by agramesh1** [comment]
Hi @gbaned and @alextp this PR seems to be stuck at internal review, can you take a look at it? Thanks.

File: tensorflow/core/common_runtime/BUILD [hunk 1]
```diff
@@ -1040,9 +1045,13 @@ cc_library(
...
     name = "mkl_tfconversion_pass",
     srcs = ["mkl_tfconversion_pass.cc"],
-    hdrs = ["mkl_tfconversion_pass.h"],
+    hdrs = [
+        "mkl_tfconversion_pass.h",
+        "//tensorflow/core/graph:mkl_graph_util_header",
+    ],
     copts = tf_copts(),
     deps = [
+        ":function",
```
Comment (jaingaurav) [inline tensorflow/core/common_runtime/BUILD:1054]: This seems to be causing a circular dependency. Try building `//tensorflow/c:c_api_test` and you should see something like:

```
.-> //tensorflow/core/common_runtime:core_cpu
|   //tensorflow/core/common_runtime:core_cpu_internal
|   //tensorflow/core/common_runtime:core_cpu_impl
|   //tensorflow/core/common_runtime:mkl_tfconversion_pass
`-- //tensorflow/core/common_runtime:core_cpu
```

**Review by jaingaurav — CHANGES_REQUESTED** [review]

File: tensorflow/core/common_runtime/BUILD (hunk 1 above)
Comment (agramesh1) [reply tensorflow/core/common_runtime/BUILD:1054]: @jaingaurav Thanks. I am not seeing this when testing CPU (with and without MKL) on Linux. Can you provide more info on what platform the test is failing.  One of my previous commits in this PR had the error on Windows builds only, but the newer commit should have fixed it.

**Pull request merged**